File: db2/scala/delete/db2_delete.scala
// ----------------------------------------------------------------------
/*
	db2_delete.scala

				May/30/2011

*/
// ----------------------------------------------------------------------
import scala.io.Source
import java.sql._

// ----------------------------------------------------------------------
object db2_delete
{

// ----------------------------------------------------------------------
def main (args: scala.Array[String])
{
	println ("*** 開始 ***")

	val id_in = args(0)

	println (id_in)

	Class.forName("COM.ibm.db2.jdbc.app.DB2Driver").newInstance()

	var conn: Connection = null

	val str_connect= "jdbc:db2:cities"

	conn = DriverManager.getConnection (str_connect,"scott","tiger")

	rdb_common.delete_proc	(conn,id_in)

	rdb_common.display_proc (conn)

	conn.close

	println ("*** 終了 ***")
}

// ----------------------------------------------------------------------
}
// ----------------------------------------------------------------------

File: db2/java/delete/db2_delete.java
// ----------------------------------------------------------------------
/*
	db2_delete.java

				May/30/2011

*/
// ----------------------------------------------------------------------
import java.sql.SQLException;
import java.sql.DriverManager;
import java.sql.Connection;

// ----------------------------------------------------------------------
public class db2_delete
{

// ----------------------------------------------------------------------
public static void main (String[] args)
	throws SQLException, ClassNotFoundException
{
	System.out.println ("*** 開始 ***");

	String	id = args[0];
	System.out.println ("\tid = " + id);

	try
		{
		Class.forName("COM.ibm.db2.jdbc.app.DB2Driver").newInstance();
		}
	catch (Exception ee)
		{
		ee.printStackTrace();
		}

	System.out.println ("*** db2_read *** aaa ***");

	Connection conn = null;

	String url = "jdbc:db2:cities";

	String user = "scott";
	String pass = "tiger";
	
	conn = DriverManager.getConnection (url,user,pass);

	System.out.println ("connected.");

	rdb_common.delete_proc	(conn,id);

	rdb_common.display_proc	(conn);

	conn.close ();

	System.out.println ("*** 終了 ***");
}

// ----------------------------------------------------------------------
}
// ----------------------------------------------------------------------
